modules/phase_field/src/materials/DerivativeParsedMaterialHelper.C

```diff
@@ -21,6 +23,8 @@ DerivativeParsedMaterialHelper::DerivativeParsedMaterialHelper(const InputParame
                                                                VariableNameMappingMode map_mode) :
     ParsedMaterialHelper(parameters, map_mode),
     //_derivative_order(getParam<unsigned int>("derivative_order"))
```

Comment:
Can this line be removed? Looks like it was superseded by line 28 below at some point.